phoenicis-tools/src/main/java/org/phoenicis/tools/system/ScreenManager.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2015-2017 PÂRIS Quentin
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package org.phoenicis.tools.system;
+
+import org.phoenicis.configuration.security.Safe;
+import org.phoenicis.entities.OperatingSystem;
+
+public class setVirtualDesktop {
```

Comment:
You need a `@Safe` annotation before the class definition, i.e.:
```java
@Safe
public class ScreenManager {
   ...
}
```

Adding `@Safe` is one of two things that are required to make this class available from the scripts :)